core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponentSpec.scala

```diff
@@ -406,11 +407,13 @@ class AttributeComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
       result
     }
 
+    val updateActionWithTempTables = (slickDataSource.createTempTable andThen updateAction andFinally slickDataSource.dropTempTable).withPinnedSession
```

Comment:
if you want to mirror `DataSource` here, it should have an initial drop: drop-create-updateAction-drop